### PR TITLE
Render time in a nice way

### PR DESCRIPTION
**Pull request by bkase**
Render time in a nice way (based on figma specs).

Wrote some unit tests -- see tests for some example outputs.

### CODE REVIEW

**Comment by psteckler** [comment]
> (based on figma specs)

Link?

**Comment by bkase** [comment]
> > (based on figma specs)
> 
> Link? It should be put in the source code, too.

Not sure if the link is permanent unfortunately (and regardless isn't yet the version of the specs we want to publish to the world). I think the tests are okay for reference for now.

**Comment by psteckler** [comment]
> Not sure if the link is permanent unfortunately

I would say it's still good to put in the PR comment, then.

File: frontend/wallet/src/common/Time.re [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  let minutes = int(Js.Date.getMinutes, date);
+
+  let nowMillis = Js.Date.getTime(now);
+  let dateMillis = Js.Date.getTime(date);
+  let epsilon = 1000. *. 60.; // 60 seconds in millis
+
+  Js.log4("Epsilon", epsilon, nowMillis, dateMillis);
+
+  if (dateMillis +. epsilon >= nowMillis) {
+    "seconds ago";
```
Comment (Schmavery) [inline frontend/wallet/src/common/Time.re]: I know I probably suggested this in person, but now that I'm thinking about it, if we're not guaranteed to be regularly updating these times, we should probably avoid a short term relative measurement like this and just remove this case.

**Comment by bkase** [comment]
> > Not sure if the link is permanent unfortunately
> 
> I would say it's still good to put in the PR comment, then.

As discussed in person, unfortunately we're not ready to share that link in a public forum yet. When it's ready, we'll make sure to update the comments around the code.

**Pull request merged**